tools/build_node.js

```diff
@@ -119,7 +119,10 @@ async function buildPackageJSON(options) {
     ".": dual("./lib/index.js"),
     "./package.json": "./package.json",
     "./lib/common": dual("./lib/common.js"),
-    "./lib/core": dual("./lib/core.js"),
+    "./lib/core": {
+      get node() { return this.require; },
```

Comment:
Whoa.  Does `JSON.stringify` automatically loop over getters or how does this work?

Reply:
Yep, it loops over all enumerable keys, and read its value by calling the getter in this case.